graphql_compiler/tests/test_testing_invariants.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 Kensho Technologies, LLC.
+"""Common GraphQL test inputs and expected outputs."""
+from inspect import getmembers, isfunction
+import unittest
+
+import graphql_compiler.tests.test_input_data as test_input_data
+
+
+def get_function_names_from_module(module):
+    return [
```

Comment:
any particular reason why this returns a `list` and the very similar function below returns a `set`?